lib/widgets/home.dart

```diff
@@ -554,11 +554,7 @@ class _SwitchAccountButton extends _MenuButton {
   const _SwitchAccountButton();
 
   @override
-  // TODO(design): choose an icon
-  IconData? get icon => null;
-
-  @override
-  Widget buildLeading(BuildContext context) => const SizedBox.shrink();
+  IconData? get icon => ZulipIcons.arrow_left_right;
```

Comment:
The commit message for the second commit now reads:

    home: Add switch account icon in main menu
    Fixes #1178.

The summary line needs to be set off from the rest of the commit message by a blank line.